### ISSUE TITLE
Lose connection when moving between mesh wifi points?

### ISSUE THREAD

**Issue by UberJason** [moritzmhmk/homebridge-camera-rpi#28]
Recently I noticed that if I put the camera on the top floor, start viewing the camera live in the Home app, and then head all the way to the bottom floor, the connection to the camera seems to drop as I reach the bottom floor and I get the message that the camera is not responding. Killing the app and re-opening it sometimes restores connection (although one time it got stuck in "someone else is viewing this camera" and I had to remove and re-add the camera to HomeKit all together).

My home uses three Google WiFi mesh routers - one per floor. I don't have a networking background, but I'm wondering if my phone moving from being connected to one wifi router to another one is causing the disconnect? Any thoughts, and possible ways to solve this problem?

**Comment by moritzmhmk** [comment]
Does the IP of your phone change when switching between the mesh routers?

**Comment by UberJason** [comment]
I don't think so, but I'm not sure - the only way I know how to check is to check the network details, and I don't know how often the details get refreshed on iOS. I checked the IP address on one floor while successfully connected to the camera, and then went to another floor (where the camera disconnected), and checked the IP address again - no change.

**Comment by moritzmhmk** [comment]
The only option would be to detect streams that crashed. It seems ffmpeg keeps streaming and thus blocks the camera.

If someone has some input on this please comment (e.g. some ffmpeg config to detect crashed streams?)
